regression/pages/studio/pages_page_studio.py

```diff
@@ -162,25 +161,28 @@ def click_hide_show_toggle(self, index=0):
             )
         )
 
-    def get_movable_default_pages_order(self):
+    def get_all_pages(self):
         """
-        Returns the order of moveable default pages
+        Returns all pages
         """
-        return self.q(
-            css='.course-nav-item.is-movable .course-nav-item-header'
-        ).text
+        temp = self.q(css='.course-tab .course-nav-item-header').text
 
-    def get_moveable_default_pages(self):
+        all_pages = temp + self.q(css='.course-tab .xblock').text
+        return all_pages
+
+    def get_all_pages_count(self):
         """
-        Returns pages which are default and moveable
+        Returns the count of all pages.
         """
-        return self.q(
-            css='.course-nav-item.is-movable .drag-handle.action'
-        ).results
+        return len(
+            self.q(
+                css='.course-tab'
+            ).results
+        )
 
     def is_page_shown(self, index=0):
```

Comment:
Could you rename the method to something like `is_page_configured_to_show` or `is_show_page_set`.. something like that?